src/components/Dialog/DialogBody/DialogBody.tsx

```diff
@@ -9,10 +9,11 @@ const b = block('dialog-body');
 export interface DialogBodyProps {
     children: React.ReactNode;
     className?: string;
+    hasDividers?: boolean;
```

Comment:
Wouldn't be `hasBorders` more precise? Divider - smth that splits the content, border - is on the edge. While it's a `DialogBody` prop, borders suits better imo.